frontend/src/components/inbox/EUCaselaw.vue

```diff
@@ -0,0 +1,145 @@
+<script lang="ts" setup>
+import { onMounted, ref } from "vue"
+import InboxList from "./shared/InboxList.vue"
+import InboxSearch from "./shared/InboxSearch.vue"
+import { Page } from "@/components/Pagination.vue"
+import { Query } from "@/composables/useQueryFromRoute"
+import { DocumentUnitSearchParameter } from "@/domain/documentUnit"
+import DocumentUnitListEntry from "@/domain/documentUnitListEntry"
+import service from "@/services/documentUnitService"
+import { ResponseError } from "@/services/httpClient"
+
+const isLoading = ref(false)
+const pageNumber = ref<number>(0)
+const itemsPerPage = 100
+const searchQuery = ref<Query<DocumentUnitSearchParameter>>()
+const currentPage = ref<Page<DocumentUnitListEntry>>()
+const searchResponseError = ref<ResponseError>()
+
+/**
+ * Searches all documentation units by given input and updates the local
+ * documentunit list entries, the currentPage for pagination and catches errors
+ * if they happen. When no results found, but a court was given, the court input
+ * is validated against the court table, in order to be able to create a new
+ * documentation unit from search the given search input, if the user wants to.
+ */
+async function search() {
+  isLoading.value = true
+
+  const response = await service.searchByDocumentUnitSearchInput({
+    ...(pageNumber.value != undefined
+      ? { pg: pageNumber.value.toString() }
+      : {}),
+    ...(itemsPerPage != undefined ? { sz: itemsPerPage.toString() } : {}),
+    ...searchQuery.value,
+    inboxStatus: "EU",
+    myDocOfficeOnly: "true",
+  })
+  if (response.data) {
+    currentPage.value = response.data
+    searchResponseError.value = undefined
+  }
+  if (response.error) {
+    searchResponseError.value = response.error
+  }
+  isLoading.value = false
+}
+
+/**
+ * Deletes a documentation unit
+ * @param {DocumentUnitListEntry} documentUnitListEntry - The entry in the list to be removed
+ */
+async function handleDelete(documentUnitListEntry: DocumentUnitListEntry) {
+  if (currentPage.value?.content) {
+    const response = await service.delete(documentUnitListEntry.uuid as string)
+    if (response.status === 200) {
+      if (currentPage.value.content.length === 1) {
+        await search()
+        return
+      }
+      const newEntries = currentPage.value.content.filter(
+        (item) => item != documentUnitListEntry,
+      )
+      currentPage.value.content = newEntries
+      currentPage.value.numberOfElements = newEntries.length
+      currentPage.value.empty = newEntries.length == 0
+    } else {
+      alert("Fehler beim Löschen der Dokumentationseinheit: " + response.data)
+    }
+  }
+}
+
+/**
+ * Updates the status from 'Fremdanlage' to 'Unveröffentlicht'
+ * @param {DocumentUnitListEntry} documentUnitListEntry - The entry in the list to be updated
+ */
+async function handleTakeOver(documentUnitListEntry: DocumentUnitListEntry) {
```

Comment:
Brauchen wir das bei EU? Alternative wäre, dass wir es quasi wie in der großen Suche machen, dass man bei Fremdanlage nur View machen kann. Allerdings sollte der Fall ja eigentlich nie auftreten (so lange Fremdanlage Periodika-spezifisch ist).
Finds aber auch nicht schlimm, wenn wir es drin lassen so just to be safe.